### ISSUE TITLE
Document setLibraryPath() which is necessary for Single Library Path Support

### ISSUE THREAD

**Issue by beberlei** [beberlei/zf-doctrine#5]
Document setLibraryPath() which is necessary for Single Library Path Support, additionally look where this is set, maybe add a hook into the application resource.

**Comment by oliverturner** [comment]
Yes please: an application resource hook that can be defined in application.ini would be very helpful
